packages/cli/src/commands/governance/approve.ts

```diff
@@ -11,37 +11,54 @@ export default class Approve extends BaseCommand {
     ...BaseCommand.flags,
     proposalID: flags.string({ required: true, description: 'UUID of proposal to approve' }),
     from: Flags.address({ required: true, description: "Approver's address" }),
+    useMultiSig: flags.boolean({
```

Comment:
Can you add a unit test?

Reply:
I did try -- its complicated because a proposal need to exist to be approve. To use the cli tooling to create a proposal it requires a json file with the proposal and url, so a bit complicated.  But it seems i can just use the contractkit directly to create one, so I will try that